### DISCUSSION TITLE
Install Issue

### DISCUSSION THREAD

**Discussion by av8rboie** [flybywiresim/aircraft#2404] [Help] [answered]
I've copied the A32NX directory to my community folder, but when I start up MSFS, and load the A320, it's still the default aircraft, None of the ecam buttons are working, the system doesn't self test on power up etc.  I do have the most recent mega livery pack in my community folder. Not sure if that could be causing an issue, but I have been testing against the default A320 NEO without any livery.

**Comment by ghost** [comment]
You need to use the installer to download the mod, as downloading directly from GitHub will not work. https://github.com/Externoak/A32NX-installer/releases/latest/download/A32NX_Downloader.zip

**Comment by PortPilot** [comment, marked as answer]
The installer will not install if the community folder is at a different drive. So I downloaded everything on the desktop and copied it to the community folder.

And suprise, it works!

regards
Jan
